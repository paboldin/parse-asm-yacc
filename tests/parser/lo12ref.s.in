	str	w1, [x2, #:lo12:.LANCHOR0]
	str	w1, [ x2, x2 ]
