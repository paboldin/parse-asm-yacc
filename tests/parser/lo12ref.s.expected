(TOKEN)	str(TOKEN)	w1(COMMA),(TOKEN) [(TOKEN)x2(COMMA),(TOKEN) #:lo12:.LANCHOR0(TOKEN)]
(TOKEN)	str(TOKEN)	w1(COMMA),(TOKEN) [(TOKEN) x2(COMMA),(TOKEN) x2(TOKEN) ]
section: name = .text, flags = x
	str	w1, [x2, #:lo12:.LANCHOR0]
	str	w1, [ x2, x2 ]
